compatibility.md

```diff
@@ -13,20 +13,13 @@ version.
 | --- | --- | --- | --- |
 | `@apollo/gateway` | 2.3.0 | 2.9.0 | `@newrelic/apollo-server-plugin@1.0.0` |
 | `@apollo/server` | 4.0.0 | 4.11.0 | `@newrelic/apollo-server-plugin@2.1.0` |
-| `@aws-sdk/client-bedrock-runtime` | 3.474.0 | 3.651.1 | 11.13.0 |
```

Comment:
hmm. I think we have to enhanced nr-versions. it needs to handle each dep not if the one file has `minSupported`. that's why the aws-sdk deps are removed @jsumners-nr

Reply:
The issue is that `@aws-sdk/smithy-client` has not set a `minSupported` value and is not defined anywhere within the tests.